oci/oci.go

```diff
@@ -577,8 +544,13 @@ func waitContainerStop(ctx context.Context, c *Container, timeout time.Duration)
 				return
 			default:
 				// Check if the process is still around
-				err := unix.Kill(c.state.Pid, 0)
-				if err == unix.ESRCH {
+				proc, err := os.FindProcess(c.state.Pid)
+				if err != nil {
+					close(done)
```

Comment:
This may be brittle on non-Linux systems.  From [the docs][1]:

> On Unix systems, FindProcess always succeeds and returns a Process for the given pid, regardless of whether the process exists. 

That doesn't talk about what happens if the process does not exist on non-Unix systems.  Maybe we'll need an `ESRCH`-style check here too.  But I'm fine punting that to follow-up work, and what you have looks right for Unix systems ([signal zero is in POSIX for this check][2]).

[1]: https://godoc.org/os#FindProcess
[2]: http://pubs.opengroup.org/onlinepubs/9699919799/functions/kill.html